README.md

```diff
@@ -320,6 +320,29 @@ const xyzOpts = {
 const xyzLayer = myMap.addLayer('xyz', xyzOpts);
 ```
 
+### Reading Features
+
+A `readFeatures` method is provided to enable reading geometry features from
+various formats like GeoJSON and WKT. It takes a format as its first parameter,
+and the input data as its second parameter.
+
+For example:
+
+```js
+// Reading features from GeoJSON.
+const geojson = {
+  type: 'Polygon',
+  coordinates: [
+    [[30, 10], [40, 40], [20, 40], [10, 20], [30, 10]]
+  ]
+}
+const features = myMap.readFeatures('geojson', geojson);
```

Comment:
Should this example show what happens to additional fields in the geoJSON?

I'd need to test, but I believe;

```javascript
const geojson = {
  type: 'Polygon',
  coordinates: [
    [[30, 10], [40, 40], [20, 40], [10, 20], [30, 10]]
  ],
  properties: {
    name: "Property Boundary"
  }
}
const features = myMap.readFeatures('geojson', geojson);

features[0].get('name') === 'Property Boundary'
```

Reply:
@symbioquine I tested this and it appears that the `name` property is NOT preserved. Although I'm not sure if that is expected behavior or not... :thinking:

Reply:
My testing suggests the geoJSON support in OL does preserve the properties - including name...

https://codesandbox.io/s/geojson-forked-8vb4on?file=/main.js

Thoughts on what might be different between you testing and mine?

Reply:
![image](https://github.com/farmOS/farmOS-map/assets/30754460/e378e93c-7440-4ad8-921f-70d57fdd011c)

https://codesandbox.io/s/geojson-forked-8vb4on?file=/main.js

Reply:
I'm not sure why it isn't working in my test. Here are the steps I took:

1. Download artifact of farmOS-map w/ new `readFeatures()` method from https://github.com/farmOS/farmOS-map/actions/runs/4917444161
2. Replaced `web/libraries/farmOS-map` in local farmOS development environment with contents of the artifact.
3. Load a farmOS page with a map, open console, and run the following code:

```
var myMap = farmOS.map.instances[0] 

const geojson = {
  type: 'Polygon',
  coordinates: [
    [[30, 10], [40, 40], [20, 40], [10, 20], [30, 10]]
  ],
  properties: {
    foo: "Property Boundary"
  }
}
const features = myMap.readFeatures('geojson', geojson);

features[0].get('foo') === 'Property Boundary' 
```

Here is a screenshot of the results in my browser:

![Screenshot from 2023-05-09 10-07-43](https://github.com/farmOS/farmOS-map/assets/95381/4d247cf1-07ca-4e00-9e05-7c10fed9c904)

Reply:
Ah ha! Great! Thanks for the explanation. I don't work directly with GeoJSON much.

I will update the bit in README.md accordingly.

Reply:
Haha, yeah I didn't see it at first either.